src/dotnet/commands/dotnet-sln/add/Program.cs

```diff
@@ -51,9 +51,13 @@ public override int Execute()
 
             var preAddProjectCount = slnFile.Projects.Count;
 
+            string relativeRoot = null;
+            if (_appliedCommand.AppliedOptions != null && _appliedCommand.AppliedOptions.Contains("-s"))
```

Comment:
Nit: we prefer checking against the long-form option so when someone reads this they don't have to go check the parser to remember what option we're checking for.

Reply:
See previous comment regarding curly braces too.